src/components/input-dropdown/input-dropdown.component.spec.ts

```diff
@@ -206,4 +212,40 @@ describe('InputDropdownComponent', () => {
         expect(openChangeSpy).toHaveBeenCalledTimes(2);
         expect(openChangeSpy).toHaveBeenCalledWith(false);
     });
+
+    it('should clear the value when allowNull is set to true', async () => {
```

Comment:
Description should be more like 'should set the selected display value to "(none)" when the clear button is clicked.' More precise is better.